tests/lib/linter/report-translator.js

```diff
@@ -1028,5 +1028,21 @@ describe("createReportTranslator", () => {
                 "Node must be provided when reporting error if location is not provided"
             );
         });
+
+        it("should throw an error if fix range is invalid", () => {
+            for (const badRange of [[0], [0, null], [null, 0], []]) {
```

Comment:
```suggestion
            for (const badRange of [[0], [0, null], [null, 0], [void 0, 1], [0, void 0], [void 0, void 0], []]) {
```

a couple of tests with explicit `undefined` in the `range` array.